packages/kobotoolbox/src/Adaptor.js

```diff
@@ -3,6 +3,14 @@ import { expandReferences } from '@openfn/language-common/util';
 
 import * as util from './Utils';
 
+/**
+ * State object
+ * @typedef {Object} KoboToolboxState
```

Comment:
This doesn't look right - `getForms` shouldn't return a `response` object? Are we doing this? If so it should be removed. And the data isn't really a response body.

I'm expecting eg `getForms` to say something like `@state data - an array of form objects`

Reply:
Fixed

Reply:
Great, thanks!